eth/main.cpp

```diff
@@ -866,7 +866,7 @@ int main(int argc, char** argv)
 				cout << "  with seed as " << seedHash << endl;
 				if (valid)
 					cout << "(mixHash = " << r.mixHash << ")" << endl;
-				cout << "SHA3( light(seed) ) = " << sha3(EthashAux::light((uint64_t)bi.number)->data()) << endl;
+				cout << "SHA3( light(seed) ) = " << sha3(EthashAux::light(bi)->data()) << endl;
```

Comment:
We already got the function overload for `BlockInfo` to `seedHash` for `light()` so I thought why not use it? Makes for smaller line length